
## 🎯 **Question**

**Does using the recommendation feature lead to higher average customer order values compared to not using the feature?**

---

## 🎯 **Hypotheses**

You are performing a **one-tailed independent t-test**, so your hypotheses would be:

### ✅ **Null Hypothesis (H₀)**

There is **no increase** in average order value for customers who used the recommendation feature:

$$
H_0: \mu_{\text{recommendation}} \leq \mu_{\text{no recommendation}}
$$

(Mean order value of recommendation users is **less than or equal to** non-users)

---

### ✅ **Alternative Hypothesis (H₁)**

Customers who used the recommendation feature have a **higher average order value**:

$$
H_1: \mu_{\text{recommendation}} > \mu_{\text{no recommendation}}
$$

---

In [7]:
import numpy as np
from scipy import stats

# Simulating sample data for two groups: 
# - `recommendation_feature`: customers who used the recommendation feature
# - `no_recommendation_feature`: customers who did not use the recommendation feature

np.random.seed(0)  # For reproducibility

# Simulating order values (in currency units)
# Customers who used the recommendation feature tend to have higher order values
recommendation_feature = np.random.normal(loc=120, scale=15, size=100)  # Mean=120, SD=15
no_recommendation_feature = np.random.normal(loc=100, scale=15, size=100)  # Mean=100, SD=15

# Conducting an independent two-sample t-test (one-tailed)
t_stat, p_value = stats.ttest_ind(recommendation_feature, no_recommendation_feature, alternative='greater')

print(f"T-statistic: {t_stat:.4f}")
print(f"P-value: {p_value:.4e}")

T-statistic: 9.0317
P-value: 7.6123e-17


Interpreting the Results

	•	If p_value < 0.05, we can reject the null hypothesis, suggesting that the recommendation feature significantly increases the average order value.
	•	If p_value >= 0.05, we fail to reject the null hypothesis, meaning there’s not enough evidence to claim the feature increases Average Order Value (AOV).

## 🎯 **Results Interpretation**

The one-tailed t-test showed:

* **T-statistic = 9.0317**
* **P-value = 7.6123e-17**

Since the p-value is far below 0.05, we reject the null hypothesis.
Customers using the recommendation feature had **significantly higher average order values** than non-users.
The recommendation feature positively impacts customer spending.